python/unitytrainers/trainer_controller.py

```diff
@@ -12,8 +12,11 @@
 from tensorflow.python.tools import freeze_graph
 from unitytrainers.ppo.trainer import PPOTrainer
 from unitytrainers.bc.trainer import BehavioralCloningTrainer
+from unitytrainers import Curriculum
 from unityagents import UnityEnvironment, UnityEnvironmentException
 
+from communicator_objects import UnityRLInitializationInput
```

Comment:
Why is this necessary?

Reply:
Woops, accidentally left that in from some experiments. Will remove.